From 550f3be28d4c44a6c2128812abc6a6cb0eb9ee78 Mon Sep 17 00:00:00 2001
From: Lars-Peter Clausen <lars@metafoo.de>
Date: Sun, 5 Sep 2010 03:21:51 +0200
Subject: [PATCH] MIPS: JZ4740: Fix i2c driver name.

The i2c driver is generic to all JZ47XX based SoCs not only the JZ4740. Reflect
this in the driver name.

Signed-off-by: Lars-Peter Clausen <lars@metafoo.de>
---
 arch/mips/jz4740/platform.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/arch/mips/jz4740/platform.c
+++ b/arch/mips/jz4740/platform.c
@@ -142,7 +142,7 @@ static struct resource jz4740_i2c_resour
 };
 
 struct platform_device jz4740_i2c_device = {
-	.name		= "jz4740-i2c",
+	.name		= "jz47xx-i2c",
 	.id		= 0,
 	.num_resources  = ARRAY_SIZE(jz4740_i2c_resources),
 	.resource	= jz4740_i2c_resources,
